pyasic/config/__init__.py

```diff
@@ -34,9 +36,11 @@ class MinerConfig:
     )
 
     def as_dict(self) -> dict:
+        """Converts the MinerConfig object to a dictionary."""
         return asdict(self)
 
     def as_am_modern(self, user_suffix: str = None) -> dict:
+        """Generates the configuration in the format suitable for AM Modern miners."""
```

Comment:
AM modern just means Antminer modern, you can change this to `...modern Antminers.`